server/belga/publish/belga_newsml_1_2.py

```diff
@@ -1040,42 +1040,32 @@ def _format_newscomponent_3_level(self, newscomponent_2_level, item):
                 SubElement(characteristics, 'Property', {'FormalName': 'maxCharCount', 'Value': '0'})
 
     def _get_author_info(self, author):
-        author_info = {
-            'initials': '',
-            'role': ''
-        }
+        author_info = {'initials': '', 'role': ''}
 
-        # get author_id
-        if type(author) is str:
+        if type(author) is dict:
+            author_info['initials'] = author.get(
+                'sub_label',
```

Comment:
Based on `SDBELGA-278`, `initials` should be username, but `sub_label` is user full name, isn't it?

Reply:
indeed, thanks